docs/news.rst

```diff
@@ -3,6 +3,155 @@
 Release notes
 =============
 
+Scrapy 1.4.0 (2017-XX-XX)
+-------------------------
+
+Scrapy 1.4 does not bring that many breathtaking new features
+but quite a few handy improvements nonetheless.
+
+Scrapy now supports anonymous FTP sessions with customizable user and
+password via the new :setting:`FTP_USER` and :setting:`FTP_PASSWORD` settings.
+**And if you're using Twisted version 17.1.0 or above, FTP is now available
+with Python 3.**
+
+Link extractors now work similarly to what a regular modern browser would
+do. Especially, leading and trailing whitespace are removed from attributes
+(think ``href="   http://example.com"``) when building ``Link`` objects.
+This whitespace-stripping also happens for ``action`` attributes with
+``FormRequest``.
+**Please also note that link extractors do not canonicalize URLs by default
+anymore.** This was puzzling users every now and then, and it's not what
+browsers do in fact, so we removed that extra transformation on extractred
+links.
+
+There's a new ``response.follow()`` shortcut for creating requests directly
```

Comment:
We could link to https://doc.scrapy.org/en/master/intro/tutorial.html#response-follow-example or https://doc.scrapy.org/en/master/topics/request-response.html#scrapy.http.TextResponse.follow